### DISCUSSION TITLE
go-micro api

### DISCUSSION THREAD

**Discussion by majest** [micro/go-micro#2065] [Q&A] [answered]
There used to be an `api` in go-micro until version 3.1.1 (https://github.com/asim/go-micro/tree/v3.1.1/api)

Are there any plans in bringing it back?

**Comment by asim** [comment, marked as answer]
No not at this time. The API package was related to v2 and used primarily with micro v2. Go Micro is now standalone and the api package does not make sense in this context. It may reemerge as something else, but its hard to justify without a real understanding of its use. What was your requirement?

**Reply by majest** [reply]
I need to expose some of the grpc services via http 

(Thanks for the quick reply)

**Reply by asim** [reply]
Something like gRPC-gateway exists for this purpose. I will take your note on board and try to reincorporate something simple for the gateway.

**Reply by majest** [reply]
Many Thanks
